test/scenarios/applicationmonitoring/without_csi.go

```diff
@@ -26,7 +26,7 @@ const (
 	sampleAppNamespace = "appmon-sample"
 )
 
-func withoutCSIDriver(t *testing.T) features.Feature {
+func appOnlyWithoutCSI(t *testing.T) features.Feature {
```

Comment:
might appMonitoringWithoutCSIOnly be a better fit?

Reply:
on it!

Reply:
`appOnlyWithoutCSI` or `appMonitoringOnlyWithoutCSI`
or if the `only` part feels a bit redundant as we are in the `applicationmonitoring` folder we can just remove that and have `appMonitoringWithoutCSI` or `applicationMonitoringWithoutCSI`

Reply:
fixed in 6f35d72f